testsuite/features/support/constants.rb

```diff
@@ -256,6 +256,11 @@
     sle-module-containers15-sp4-updates-x86_64
     sle-module-basesystem15-sp4-updates-x86_64
     sle-module-server-applications15-sp4-updates-x86_64
+    sle15-sp4-installer-updates-x86_64
+    sle-module-desktop-applications15-sp4-pool-x86_64
+    sle-module-desktop-applications15-sp4-updates-x86_64
+    sle-module-devtools15-sp4-pool-x86_64
+    sle-module-devtools15-sp4-updates-x86_64
```

Comment:
I am wondering if we can spare some of them, like the development tools

Reply:
I believe I added all those that were necessary for the features in the testsuite, so they would be the minimum required
I'll double check on a test branch just in case and document it in the PR description

Reply:
DevTools are sadly a parent of one of the channels we need, so it's a requirement we can't remove.

Reply:
ok